README.Rmd

```diff
@@ -345,3 +346,20 @@ ggplot(H_CARI_streams) +
   geom_sf()+
   theme_bw()
 ```
+
+# Stations
+
+## Stations from IEP's long term monitoring surveys
+
+These data were collated from the discretewq package, zooper package, and other IEP data published on EDI. Code was modified from:
+https://deltascience.shinyapps.io/monitoring/ 
```

Comment:
Probably better to link to the actual code (https://github.com/sbashevkin/stations) although you can keep the shiny app link as well if you think it's helpful

Reply:
Done!